### DISCUSSION TITLE
How to set the `-local` option for `goimports`?

### DISCUSSION THREAD

**Discussion by bravoecho** [w0rp/ale#3977] [Q&A - Ask for help with problems] [answered]
When I use `goimports` as a fixer for Go, I would like to specify the `-local` option. Ideally, if the file belongs to the local module, it would be great if that were done automatically. Is either of these possible? Thank you!

**Comment by hsanson** [comment, marked as answer]
There seems to be a `g:ale_go_goimports_options` setting you can use to pass options to the tool:

```
let g:ale_go_goimports_options = '-local'
```

Note the option does not seem to be documented but the implementation does use it.

**Reply by bravoecho** [reply]
Awesome, thank you, that works! 🙂
